### PR TITLE
Connect API

### PR DESCRIPTION
**Pull request by melaniesigrid**
- Connect app to API using Thunks and AXIOS

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { combineReducers, createStore, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
+import thunk from 'redux-thunk';
 import booksReducer from './books/books';
```
Comment (FrederickMih) [inline src/redux/configureStore.js:4]: - [x] Hi @melaniesigrid, please let's make sure that after refactoring our app to use the Bookstore API, the AddBook and Remove book feature should work the same. When I try to post a book to the server, I get a console error. Our book data is not persisted correctly to the server. You may consider looking critically into the importation `import booksReducer from './books/books';`

**Review by FrederickMih — CHANGES_REQUESTED** [review]
# Hi @melaniesigrid , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (melaniesigrid) [reply src/redux/configureStore.js:4]: Thank you for your review! I have corrected the issues sending and receiving from the API.

**Review by EricMbouwe — APPROVED** [review]
### Hi @melaniesigrid, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**